### ISSUE TITLE
[gui error report] Error: Initializing daemon...

### ISSUE THREAD

**Issue by AndyinHawick** [ipfs/ipfs-desktop#2379]
👉️ Please describe what you were doing when this error happened.

I have been banging my head against this one for a while. I have tried upgrading and downloading various things to try and solve the problem IPFS Desktop has repeatedly told me that it had downloaded an update and Restart to install it, which I did … but it didn't!  I manually downloaded the new version of Desktop from this site and then got this error, which is the same as I've been getting with Terminal when trying to manually start the daemon.

Am I missing something? If so, what?

**Specifications**

- **OS**: darwin 22.2.0
- **IPFS Desktop Version**: 0.25.0
- **Electron Version**: 19.1.9
- **Chrome Version**: 102.0.5005.167

**Error**

```
Error: Initializing daemon...
Kubo version: 0.17.0
Repo version: 12
System version: amd64/darwin
Golang version: go1.19.1

Error: Your programs version (12) is lower than your repos (13).
Please update ipfs to a version that supports the existing repo, or run
a migration in reverse.

See https://github.com/ipfs/fs-repo-migrations/blob/master/run.md for details.

    at errorTemplate (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/src/daemon/migration-prompt.js:78:49)
    at Object.loadWindow (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/src/daemon/migration-prompt.js:129:26)
    at startIpfsWithLogs (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/src/daemon/daemon.js:191:25)
    at process.processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async startDaemon (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/src/daemon/daemon.js:216:27)
    at async startIpfs (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/src/daemon/index.js:48:17)
    at async setupDaemon (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/src/daemon/index.js:116:3)
    at async run (/Applications/IPFS Desktop.app/Contents/Resources/app.asar/src/index.js:85:5)
```

**Comment by AndyinHawick** [comment]
I have now got IPFS running by downgrading to v0.16.0 (rather than v0.17.0 with a manual update via Terminal.

**Comment by AndyinHawick** [comment]
Hi Nishant

I am fairly certain that I didn't use a package manager to install any of the components. I have mainly used the UI of HA itself to install updates noted although when I was having problems I linked through to the GitHub pages and downloaded packages from there. Some of the terminology I am unfamiliar with and at times I found myself going around in circles, following the instructions only to land back at the same place. Eventually, I worked out which component needed downgrading and found detailed instruction of how to do that and successfully implemented the downgrade to get things working again.

Part of the complication may have been that I had HA successfully (but unstably) running on an old MacBook (Pro 5,5) and used a backup to install the configurations onto a Pi (3) set up solely for the purposes of running HA. I gather that there are various possible configurations for running HA and am not entirely clear as to how these two set-ups would be described let alone what differences there are and whether alternative versions are needed in each case.

I hope that helps. But do come back to me with specific questions if they might help with your debugging.

Many thanks
	Andy
